includes/services/GoogleMaps3/jquery.googlemap.js

```diff
@@ -513,6 +513,10 @@
 				mapOptions.mapTypeControl = false;
 			}
 
+			if (options.scrollwheelzoom === true) {
+				mapOptions.gestureHandling = 'greedy';
+			}
```

Comment:
@hidrarga please beware that the style guide requires if statement curly brackets (I added them here as they where missing)